stores/sql/mysql/migrations/main/migration_00030_remove_contract_sets.sql

```diff
@@ -0,0 +1,13 @@
+-- drop default contract set from settings
+UPDATE settings SET value=JSON_REMOVE(value, '$.defaultContractSet') WHERE `key`="upload";
+
+-- remove contract set reference from slabs table
+ALTER TABLE slabs DROP FOREIGN KEY fk_slabs_db_contract_set;
+ALTER TABLE slabs DROP COLUMN db_contract_set_id;
+
+-- update usability
+UPDATE contracts SET usability = CASE WHEN id IN (SELECT db_contract_id FROM contract_set_contracts) THEN 2 ELSE 1 END;
```

Comment:
I think this update's subquery performs a bit more work than necessary.
So if you haven't gone through the trouble of manually testing this, the following might be more efficient since it only select one id per contract to update instead of all the ids.

`UPDATE contracts SET usability = CASE WHEN EXISTS (SELECT 1 FROM contract_set_contracts WHERE db_contract_id = contracts.id) THEN 2 ELSE 1 END;`